user-service/internal/models.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import "github.com/go-redis/redis/v8"
+
+func GetMoney(username string) int {
+
+	val, err := Rdb.Get(Ctx, username).Int()
+
+	if err == redis.Nil {
+		return -1

```

Comment:
I suppose this error happens when the user is non-existent. You should probably return 0 in that case.